pycobertura/reporters.py

```diff
@@ -54,6 +54,16 @@ def get_report_lines(self):
 
         return lines
 
+    def per_file_stats(self, lines):
+        rows = []
+        items = lines.items()
+        number_of_files = len(self.cobertura.files())
+        for i in range(number_of_files):
+            if i < number_of_files:
```

Comment:
Won't `i` always be less than `number_of_files` here? 🤔

Reply:
Should be addressed in 0f5035c81f0021caaaffa9e953480d7f00c76f81. Though what confuses me a little is that for the diff the formula is:
```
number_of_files = len(self.differ.files()) - 1
```

, whereas for the plain show the formula is:

```
number_of_files = len(self.cobertura.files())
```

Note the missing -1 in the second case.
So could one of you, @aconrad or @notpushkin, check that this is really correct.